hack/update-crds.sh

```diff
@@ -25,4 +25,13 @@ fi
 
 operator-sdk generate k8s
 
-operator-sdk generate openapi
+operator-sdk generate crds
+
+go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
+
+./bin/openapi-gen -i ./pkg/apis/csi/v1 \
```

Comment:
is there a reason not to use [`--logtostderr=true`](https://sdk.operatorframework.io/docs/upgrading-sdk-version/version-upgrade-guide/#v017x)?

Reply:
It's the default value.
Adding this flag doesn't contribute IMO.
Also, adding every flag with its default value makes the command very long and confuses the readers.

Reply:
ok, before we resolve this, could you just show here how you know its default value?

Reply:
orian@OrianZinge-5GQ4:/mnt/c/Windows/system32$ openapi-gen --help
Usage of openapi-gen:
...
      --logtostderr                      log to standard error instead of files (default true)
...

Reply:
thanks, let's hope the default won't change